jenkins/databricks/test.sh

```diff
@@ -69,10 +69,9 @@ IS_SPARK_311_OR_LATER=0
 
 
 # TEST_MODE
-# - IT_ONLY
-# - CUDF_UDF_ONLY
-# - ALL: IT+CUDF_UDF
-TEST_MODE=${TEST_MODE:-'IT_ONLY'}
+# - DEFAULT: all tests except cudf_udf tests
+# - CUDF_UDF_ONLY: cudf_udf tests only, requires extra conda cudf-py lib
```

Comment:
seems like we should add the iceberg and delta lake as comments here

Reply:
yeah, I will update this one